src/Core/Search/Filters.php

```diff
@@ -39,6 +39,9 @@ class Filters extends ParameterBag implements SearchCriteriaInterface
     /** @var string */
     protected $filterId = '';
 
+    /** @var bool */
+    protected $isPersisted = true;
```

Comment:
1. does this method tells if filters should be persisted or not? its a bit unclear whether it is a setting or a just a getter. Its more intuitive to think that "isPersisted" will tell you if filters was parsisted already, but not so obvious that they will actually impact the persistence of the filters. Just at first glance... Idk if it makes sense, I might be overthinking :man_shrugging: 
2. so did you remove the persistence of combinations filters?

Reply:
Yeah you're right!